src/Helper/InternalHelper.php

```diff
@@ -0,0 +1,63 @@
+<?php
+declare(strict_types=1);
+
+namespace PHPSu\Helper;
+
+use PHPSu\Controller;
+use PHPSu\Exceptions\CommandExecutionException;
+use PHPSu\Exceptions\EnvironmentException;
+use PHPSu\Process\CommandExecutor;
+use PHPSu\Tools\EnvironmentUtility;
+
+final class InternalHelper
+{
+
+    public function getCurrentPHPSUVersion(): string
```

Comment:
This looks better for me. What do you think @ChrisB9?

```suggestion
    public function getCurrentPHPSUVersion(): string
    {
      return $this->getPhpSuVersionFromVendor() ?? $this->getPhpSuVersionFromGit() ?? 'development';
    }
```